tor.mk

```diff
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS  += tor
+DOWNLOAD     += https://dist.torproject.org/tor-$(TOR_VERSION).tar.gz
+TOR_VERSION  := 0.4.3.5
+DEB_TOR_V    ?= $(TOR_VERSION)
+
+tor-setup: setup
+	$(call EXTRACT_TAR,tor-$(TOR_VERSION).tar.gz,tor-$(TOR_VERSION),tor)
+
+ifneq ($(wildcard $(BUILD_WORK)/tor/.build_complete),)
+tor:
+	@echo "Using previously built tor."
+else
+tor: tor-setup
```

Comment:
Does tor link to openssl, libevent, xz, and/or zstd? If so, why aren't they a required target here, and if not, why are they listed as Depends in the control file?